### PR TITLE
reduce delay of greeting message on login and refresh

### PR DESCRIPTION
**Pull request by Aryan-9211**
Navbar GitHub login button flickering issue for Signed In user #182

The delay might be become it waits for the dom to load so instead made the function to run as soon as the
javascript loads to reduce load time.

No the development is not tested.

### CODE REVIEW

**Comment by rajat-mehra05** [comment]
You forgot to run `npm run check` before pushing the PR. Please look at the errors in detail on what's blocking it. :)

File: js/user.js [hunk 1]
```diff
@@ -24,20 +24,18 @@ const setUserGreeting = (username, firstName) => {
   }
 };
 
-const fetchData = () => {
-  fetch('https://api.realdevsquad.com/users/self', {
+async function fetchData() {
```
Comment (rohan09-raj) [inline js/user.js:27]: Use `arrow` function here.

File: js/user.js [hunk 2]
```diff
@@ -24,20 +24,18 @@ const setUserGreeting = (username, firstName) => {
...
-};
+  });
+  const json = await res.json();
+  if (json.incompleteUserDetails) {
+    return window.location.replace('https://my.realdevsquad.com/signup');
+  }
+  setUserGreeting(json.username, json.first_name);
+}
 
-window.addEventListener('DOMContentLoaded', fetchData);
```
Comment (rohan09-raj) [inline js/user.js:41]: Why you have removed this line ??  
Read up more here:
https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event

File: .github/Pull_Request_Template.md [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
 
 ### Before / After Change Screenshots
+
 For visual or interaction changes. Please provide a video / screenshot.
 
 ##### Example:
-Before : 
+
+Before :
 Image/Video
```
Comment (rohan09-raj) [inline .github/Pull_Request_Template.md:23]: Revert these changes please and resolve the merge conflicts if you face any.

**Review by rohan09-raj — CHANGES_REQUESTED** [review]

**Review by rajat-mehra05 — APPROVED** [review]
Thanks Aryan for these changes. Looks good now. :)

**Review by rajat-mehra05 — APPROVED** [review]
